src/TrafficSign/TrafficSign.types.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Properties of the TrafficSign component
+ */
+
+export type TrafficSignProps = {
+  /**
+   * The type of the traffic sign
+   */
+  type:
+    | "barrier"
+    | "barrier2S"
+    | "beaconArrowL"
+    | "beaconArrowR"
+    | "beaconHatchedWide"
+    | "beaconHatchedWideL"
+    | "beaconHatchedWideR"
+    | "beaconL"
+    | "beaconL2S"
+    | "beaconR2S"
+    | "beaconR"
+    | "caution"
+    | "children"
+    | "crossroads"
+    | "curveL"
+    | "curveR"
+    | "cyclistsCrossing"
+    | "cyclistsCrossingL"
+    | "cyclistsCrossingR"
+    | "deadEnd"
+    | "directionBoardL"
+    | "directionBoardR"
+    | "directionBoardWideL"
+    | "directionBoardWideR"
+    | "diversionL"
+    | "diversionR"
+    | "endOfLimitations"
+    | "exitHighway"
+    | "exitInnerCityRoad"
+    | "exitLimitedAccessRoad"
+    | "fileUnknown"
+    | "giveWay"
+    | "highway"
+    | "highwayEnd"
+    | "icy"
+    | "limitedAccessRoad"
+    | "limitedAccessRoadEnd"
+    | "livingStreet"
+    | "livingStreetEnd"
+    | "minimumSpeed"
+    | "minimumSpeedEnd"
+    | "narrowRoad"
+    | "narrowRoadL"
+    | "narrowRoadR"
+    | "noEntry"
+    | "noOvertaking"
+    | "noOvertakingEnd"
+    | "noOvertakingTrucks"
+    | "noOvertakingTrucksEnd"
+    | "noTraffic"
+    | "oneWayLeft"
+    | "oneWayRight"
+    | "oneWayStraight"
+    | "parkingGarage"
+    | "parkingLot"
+    | "passL"
+    | "passR"
+    | "pedestrianCrossing"
+    | "pedestrians"
+    | "priorityRoad"
+    | "priorityRoadEnd"
+    | "rightOfWay"
+    | "roadWorks"
+    | "roundabout"
+    | "sCurveL"
+    | "sCurveR"
+    | "slipperyRoad"
+    | "speedLimit"
+    | "speedLimitEnd"
+    | "speedLimitZone"
+    | "speedLimitZoneEnd"
+    | "steepAscent"
+    | "steepDescent"
+    | "stop"
+    | "straightOrLeft"
+    | "straightOrRight"
+    | "trafficLight"
+    | "trafficQueues"
+    | "tunnel"
+    | "turnAheadL"
+    | "turnAheadR"
+    | "turnL"
+    | "turnR"
+    | "unevenRoad"
+    | "urbanArea"
+    | "urbanAreaEnd"
+    | "warningPedestrianCrossing"
+    | "weightLimit"
+    | "widthLimit";
+  /**
+   * The coordinates of the sign in Fr0 [x,y]
+   */
+  points: number[];
```

Comment:
Is this typed correctly? number[] allows any length array, e.g. [1, 2, 3, 4]. Are you after [1, 2] or [ [1,2], [3,4] ]?

```
type option1 = number[]
type option2 = [number, number]
type optionm3 = [number, number][]
```

Reply:
Typed wrong, intended was `[number. number]`